controlplane/kubeadm/internal/cluster_test.go

```diff
@@ -21,15 +21,28 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
 
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 )
 
+var (
```

Comment:
Is there a reason to do this over importing `. "github.com/onsi/gomega"`?

Reply:
This came in via a cherry-pick from New Relic's branch. 

I've been avoiding the `.` import, because `And`, `Or`, and `Not` conflict with filters defined in this package.

I suspect the idea here was to provide aliasing to the `gomega` methods used to avoid having to type out the full names.

Reply:
Ah, the filters. Of course. 👍

Reply:
The dot import leads to a collision between the `Or` function in gomega and the `Or` function in cluster.go.

Reply:
I'd rather be consistent with the entire codebase and use the dot import, change the name of the filters in `cluster.go`